ansible/roles/operator-pipeline/templates/openshift/tasks/verify-ci-results.yml

```diff
@@ -18,10 +19,14 @@ spec:
       workingDir: $(workspaces.results.path)
       script: |
         #! /usr/bin/env bash
-        # Did the tests pass?
+
         # did preflight/ CI have the latest version?
-        # If results exists, store them in the workspace
-        # In case of failure- comment on the PR with tailored support link
+        PREFLIGHT_VERSION=$(cat PREFLIGHT_RESULTS | jq '.test_library.version' | grep .)
+        if [$PREFLIGHT_VERSION < $(params.preflight_min_version)]; then
```

Comment:
Are you sure, that it's a correct bash syntax? I cannot force it to work. Also, I'm sure this comparison won't work for dot separated versions like `4.8`.

Reply:
Updated!

Reply:
I really like the new solution!